CHANGELOG.md

```diff
@@ -2,7 +2,19 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
-### [4.0.0](https://github.com/fiatconnect/fiatconnect-types/compare/v3.3.0...v4.0.0) (2022-05-25)
+## [4.1.0](https://github.com/fiatconnect/fiatconnect-types/compare/v3.3.0...v4.1.0) (2022-06-06)
```

Comment:
```suggestion
## [4.1.0](https://github.com/fiatconnect/fiatconnect-types/compare/v4.0.0...v4.1.0) (2022-06-06)
```

Reply:
good catch. Not sure why semantic-release used 3.3.0 here

Reply:
I think this was because v4.0.0 was tagged on a commit on a non-main branch. I'll fix it when I publish this